### PR TITLE
Northwind v2

### PR DESCRIPTION
**Pull request by ian-neo4j**
Changes to copy and structure.

### CODE REVIEW

**Comment by ian-neo4j** [comment]
Copy and structure changes

**Comment by ian-neo4j** [comment]
Copy and structure changes

**Comment by ian-neo4j** [comment]
Moved sentences to new lines

File: documentation/northwind.adoc [hunk 1]
```diff
@@ -1,151 +1,178 @@
...
-A graph is made up of nodes, and relationships each with type(s) and properties.
+The nodes and relationships in your graph can have *Types* and *Properties*. 
 
 === Nodes and Properties
 
-
-Nodes can be tagged with _Labels_, representing their different roles in your domain. (`Supplier`, `Employee`, `Customer`). 
-They can hold any number of key-value pairs as _Properties_. (`name:"Jane"`).
+Nodes can be tagged with *Labels*, representing their different roles in your dataset (e.g. `Supplier`, `Employee`, `Customer`).
+They can also hold any number of key-value pairs as *Properties*. (`name:"Jane"`).
```
Comment (jexp) [inline documentation/northwind.adoc:52]: we should probably change that to a entry from the dataset e.g. a productname

File: documentation/northwind.adoc [hunk 2]
```diff
@@ -1,151 +1,178 @@
...
+=== Schema free
+
+In a relational database, you must define your schema (the structure of the data). In a graph, your data is managed without restricting it to a pre-defined schema. This allows a very flexible way of thinking and evolving it.
 
-This can have exciting and powerful implications for what you can do with your data.
 
 == Nodes, properties, relationships
 
-A graph is made up of nodes, and relationships each with type(s) and properties.
+The nodes and relationships in your graph can have *Types* and *Properties*.
```
Comment (jexp) [inline documentation/northwind.adoc:47]: casing consistency?

File: documentation/northwind.adoc [hunk 3]
```diff
@@ -1,151 +1,178 @@
...
 image::https://github.com/neo4j-graph-examples/northwind/raw/main/documentation/img/model.svg[]
 
-In relational databases, references to other rows and tables computed at query time through the use of joins.
-These can be complex, slow, and expensive.
+In relational databases, the relationships between rows and other tables are not stored. 
+Instead, you must write queries that use a `join`. These operations can be complex, slow, and expensive. 
 
-In the graph property model, those relationships are stored in the database. 
-Queries that follow relationships are then simple and fast.
+In a graph, you get them insantly and for free. This has powerful implications for many data-driven applications.
```
Comment (jexp) [inline documentation/northwind.adoc:38]: Too fluffy markety

File: documentation/northwind.adoc [hunk 4]
```diff
@@ -1,151 +1,178 @@
...
-4.  Query patterns in your data
 
 == What is a graph?
 
-A graph database stores entities (nodes) and their relationships instead of tables, or documents. 
-Data is stored just like you might sketch ideas on a whiteboard. 
-Your data is managed without restricting it to a pre-defined schema, allowing a very flexible way of thinking and evolving it.
+=== Concepts
+
+A graph database stores *nodes* (`Customer`, `Product`, etc) and their *relationships* (e.g. `Customer PURCHASED Product`).
```
Comment (jexp) [inline documentation/northwind.adoc:29]: Use a relationship from the actual graph model

File: documentation/northwind.adoc [hunk 5]
```diff
@@ -1,151 +1,178 @@
...
+A graph database stores *nodes* (`Customer`, `Product`, etc) and their *relationships* (e.g. `Customer PURCHASED Product`). 
+
+Unlike other types of database, which may use tables or documents, in a graph data is stored just like you might sketch ideas on a whiteboard.
 
 image::https://github.com/neo4j-graph-examples/northwind/raw/main/documentation/img/model.svg[]
 
-In relational databases, references to other rows and tables computed at query time through the use of joins.
-These can be complex, slow, and expensive.
+In relational databases, the relationships between rows and other tables are not stored. 
+Instead, you must write queries that use a `join`. These operations can be complex, slow, and expensive.
```
Comment (jexp) [inline documentation/northwind.adoc:36]: this bit (is actually also true for non-relational dbs), and saying they are not stored opens up a can of worms as people will state that "of course foreign keys are stored" ...
Question is what do we want to point out?
the need to add explicit join syntax to your queries (which will be alleviated by SQL-PG)
the issue that joins have to be computed each time a query runs?

**Comment by ian-neo4j** [comment]
Hopefully resolved all inconsistencies.

**Comment by ian-neo4j** [comment]
Relocated northwind.adoc file to /workspace

**Pull request merged**